src/ports/postgres/modules/deep_learning/madlib_keras_fit_multiple_model.sql_in

```diff
@@ -879,7 +914,7 @@ num_classes         | 3
 -# Define and load model architecture.  Use Keras to define
 the model architecture with 1 hidden layer:
 <pre class="example">
-import keras
+from tensorflow import keras
```

Comment:
we should use tensorflow.keras in the next two lines as well

Reply:
fixed